src/components/ElvaYBPractice2/styles/MainComponent.styled.ts

```diff
@@ -0,0 +1,258 @@
+import styled from 'styled-components'
+import { Slider, Switch } from 'antd'
+
+import BackgroundPic from '../../../assets/ElvaYongbin/BackgroundPic.svg'
+import PatternCirclePic from '../../../assets/ElvaYongbin/images/pattern-circles.svg'
+import SliderIcon from '../../../assets/ElvaYongbin/images/icon-slider.svg'
+
+export const MainComponent = styled.div`
+z-index: 2;
+border-radius: 15px;
+width: max-content;
+height: auto;
+display: flex;
+padding-top: 30px;
+background-color: #ffffff;
+align-items: center;
+justify-content: center;
+flex-direction: column;
+box-shadow: 0px 3px 10px rgb(0 0 0 / 0.2);
+@media only screen and (max-width: 600px) {
+  margin-top: 180px;
+  height: 60vh;
+  width: 55vw;
```

Comment:
consider increasing the width, according to FE mentor challenge, there isn't as much space as u have now

Reply:
can also consider adding min width, if you dont want this to happen. But the value of min width has to be very low.

Reply:
Resolved in commit c5c8c19